### PR TITLE
feat: improve nft collectible

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes # MF-790

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        { title: 'Creator Royalties', value: _asset.contract?.creatorEarning ?? '0' },
+        { title: 'OpenSea Platform costs', value: '2.5%' },
+    ]
+    return (
+        <div className={classes.wrapper}>
+            {infoConfigMapping.map((x) => {
+                return (
+                    <div key={x.title} className={classes.listItem}>
+                        <Typography className={classes.textBase}>{x.title}</Typography>
+                        <div className={classes.listItemContent}>{x.value}</div>
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx]: Use `<Typography />` to wrap texture.

File: packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+interface NFTOfferCardProps {
+    offer: NonFungibleTokenOrder<ChainId, SchemaType>
+}
+
+export function NFTOfferCard(props: NFTOfferCardProps) {
+    const { offer } = props
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    // if (!offer.value || offer.loading) return <Skeleton width="100%" height={60} />
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx]:
```suggestion
```

File: packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    return (
+        <div className={classes.wrapper}>
+            <OpenSeaIcon width={24} height={24} />
+            <div className={classes.offerDetail}>
+                <div className={classes.flex}>
+                    {<img className={classes.currencyIcon} src="" alt="" /> || <ETHIcon width={20} height={20} />}
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx]:
```suggestion
                    {<img className={classes.currencyIcon} src="" alt="" />
```

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
+}
+
+export function NFTPriceCard(props: NFTPriceCardProps) {
+    const { asset } = props
+    const { classes } = useStyles()
+    if (!asset.value || asset.loading) return <Skeleton width="100%" height={146} />
+    const _asset = asset.value
+    console.log(_asset, 'ggg')
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]:
```suggestion
```

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        display: 'flex',
+        gap: 4,
+    },
+}))
+
+interface NFTPriceCardProps {
+    asset: {
+        loading?: boolean
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx:48]:
```suggestion
    loading?: boolean
    asset?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
```

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        (_asset.paymentTokens?.length
+            ? _asset.paymentTokens[0].logoURL
+            : 'https://static.debank.com/image/token/logo_url/eth/935ae4e4d1d12d59a99717a24f2540b5.png')
+
+    return (
+        <div className={classes.wrapper}>
+            <div className={classes.header}>
+                <Typography className={classes.textBase}>Price</Typography>
+                <Typography className={classes.textBase}>
+                    Time left <strong>-</strong> h <strong>-</strong> m
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: i18n

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    const { asset } = props
+    const { classes } = useStyles()
+    if (!asset.value || asset.loading) return <Skeleton width="100%" height={146} />
+    const _asset = asset.value
+    console.log(_asset, 'ggg')
+    const priceTokenImg =
+        _asset.priceInToken?.token.logoURL ||
+        (_asset.paymentTokens?.length
+            ? _asset.paymentTokens[0].logoURL
+            : 'https://static.debank.com/image/token/logo_url/eth/935ae4e4d1d12d59a99717a24f2540b5.png')
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: If no payment token is available, then hide the payment token part in UI.

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+                <Typography className={classes.textBase}>
+                    Time left <strong>-</strong> h <strong>-</strong> m
+                </Typography>
+            </div>
+            <div className={classes.priceZone}>
+                <img width={48} height={48} src={priceTokenImg} />
+                <Typography className={classes.priceText}>{_asset.priceInToken?.amount ?? '-'}</Typography>
+            </div>
+            <div className={classes.offerBox}>
+                <Typography className={classes.textBase}>Top Offer</Typography>
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: i18n

File: packages/mask/src/components/shared/NFTCard/NFTRank.tsx [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+interface NFTRankProps {
+    providerType?: SourceType
+    rank: number
+}
+
+export function NFTRank(props: NFTRankProps) {
+    const { providerType, rank } = props
+    // todo: multiple style to provider
+    const { classes } = useStyles()
+    const _providerType = providerType || SourceType.Gem
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTRank.tsx]:
```suggestion
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Collectible.tsx [hunk 10]
```diff
@@ -138,20 +128,21 @@ export function Collectible(props: CollectibleProps) {
...
+        setProvider(v)
     }, [])
     // #endregion
 
     // #region provider switcher
     const CollectibleProviderSwitcher = useSwitcher(provider, setProvider, supportedProvider, resolveSourceName, true)
     // #endregion
-
+    const [currentTab, onChange, tabs, setTab] = useTabs('about', 'details', 'offers', 'activity')
+    console.log(asset, 'asset')
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Collectible.tsx]:
```suggestion
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/ActivityTab.tsx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+                </div>
+            )
+        return (
+            <CollectibleTab>
+                <div className={classes.body} style={{ justifyContent: 'unset' }}>
+                    <>
+                        {_events.map((x, idx) => (
+                            <NFTActivityCard type={ActivityType.Transfer} key={idx} activity={x} />
+                        ))}
+                    </>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/ActivityTab.tsx]:
```suggestion
                    {_events.map((x, idx) => (
                        <NFTActivityCard type={ActivityType.Transfer} key={idx} activity={x} />
                    ))}
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+                </div>
+            )
+        return (
+            <CollectibleTab>
+                <div className={classes.body} style={{ justifyContent: 'unset' }}>
+                    <>
+                        {_orders?.map((x, idx) => (
+                            <NFTOfferCard key={idx} offer={x} />
+                        ))}
+                    </>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx]:
```suggestion
                    {_orders?.map((x, idx) => (
                        <NFTOfferCard key={idx} offer={x} />
                    ))}
```

File: packages/web3-providers/src/opensea/index.ts [hunk 13]
```diff
@@ -430,6 +431,7 @@ export class OpenSeaAPI implements NonFungibleTokenAPI.Provider<ChainId, SchemaT
             }),
             chainId,
         )
+        console.log(response, 'orders123')
```
Comment (guanbinrui) [inline packages/web3-providers/src/opensea/index.ts]:
```suggestion
```

File: packages/mask/shared-ui/locales/en-US.json [hunk 14]
```diff
@@ -694,12 +696,14 @@
...
     "plugin_collectible_date": "Date",
     "plugin_collectible_quantity": "Quantity",
+    "plugin_collectible_expires_in": "Expires in",
     "plugin_collectible_expiration": "Expiration",
     "plugin_collectible_no_offers": "No Offers",
     "plugin_collectible_no_listings": "No Listings",
     "plugin_collectible_base": "Base",
     "plugin_collectible_properties": "Properties",
     "plugin_collectible_about": "About",
+    "plugin_collectible_nft_activity_empty": "This NFT didn't have any activity",
```
Comment (guanbinrui) [inline packages/mask/shared-ui/locales/en-US.json]:
```suggestion
    "plugin_collectible_nft_activity_empty": "This NFT didn't have any activity.",
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+    emptyText: {
+        fontSize: 14,
+        color: theme.palette.maskColor.second,
+    },
+}))
+
+export function OffersTab() {
+    const { classes } = useStyles()
+    const { orders } = CollectibleState.useContainer()
+    const _orders = orders.value?.data ?? []
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx]:
```suggestion
    const _orders = orders.value?.data ?? EMPTY_LIST
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/ActivityTab.tsx [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+    },
+}))
+
+export interface ActivityTabProps {}
+
+export function ActivityTab(props: ActivityTabProps) {
+    const { classes } = useStyles()
+    const { events } = CollectibleState.useContainer()
+    const { t } = useI18N()
+    const _events = events.value?.data ?? []
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/ActivityTab.tsx]:
```suggestion
    const _events = events.value?.data ?? EMPTY_LIST
```

File: packages/mask/src/components/shared/NFTCard/NFTPropertiesCard.tsx [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+        <div className={classes.wrapper}>
+            <div className={classes.titleBox}>
+                <Typography className={classes.title}>{t('plugin_collectible_properties')}</Typography>
+                <div className={classes.rankBox}>
+                    {/* gem rank */}
+                    <NFTRank rank={1} />
+                </div>
+            </div>
+            <div className={classes.content}>
+                {_asset.traits?.map((x: any) => {
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPropertiesCard.tsx]: No need to use `any` here. We have defined it in the type: `NonFungibleAsset`.

File: packages/mask/src/components/shared/NFTCard/NFTDescription.tsx [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+        padding: 12,
+    },
+    textContent: {
+        fontSize: 14,
+        lineHeight: '18px',
+        color: theme.palette.maskColor.second,
+    },
+}))
+interface NFTDescriptionProps {
+    asset: any
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTDescription.tsx]: Don't use `any`.

File: packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+        display: 'flex',
+        gap: 6,
+    },
+}))
+
+interface NFTInfoCardProps {
+    asset: {
+        loading?: boolean
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx]:
```suggestion
    asset: AsyncState<Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>>
```

File: packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+export function NFTInfoCard(props: NFTInfoCardProps) {
+    const { asset } = props
+    const { classes } = useStyles()
+    const { Others } = useWeb3State()
+    if (!asset.value || asset.loading) return <Skeleton width="100%" height={172} />
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx]: Do we need to handle the error state?

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx (hunk 5 above)
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]:
```suggestion
    asset: AsyncState<Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>>
```

File: packages/mask/src/components/shared/NFTCard/NFTPropertiesCard.tsx [hunk 21]
```diff
@@ -0,0 +1,92 @@
...
+    traitsTitle: {
+        fontWeight: 400,
+        color: theme.palette.maskColor.second,
+    },
+}))
+interface NFTPropertiesCardProps {
+    asset: {
+        loading?: boolean
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPropertiesCard.tsx]:
```suggestion
    asset: AsyncState<Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>>
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/AboutTab.tsx [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+        width: '100%',
+        marginBottom: 16,
+    },
+}))
+
+export interface AboutTabProps {
+    asset: {
+        loading?: boolean
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/AboutTab.tsx]:
```suggestion
    asset: AsyncState<Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>>
```

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/DetailTab.tsx [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+        width: '100%',
+        marginBottom: 24,
+    },
+}))
+
+export interface DetailTabProps {
+    asset: {
+        loading?: boolean
+        value?: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+    }
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/DetailTab.tsx]:
```suggestion
    asset: AsyncState<Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>>
```

File: packages/mask/src/components/shared/NFTCard/NFTBasicInfo.tsx [hunk 24]
```diff
@@ -0,0 +1,132 @@
...
+                            </div>
+                        )
+                    })}
+                </div>
+                <NFTCardStyledAssetPlayer url={resourceUrl} classes={classes} isNative={false} />
+            </div>
+            <Typography className={classes.nameSm}>
+                {_asset.metadata?.name.includes('#')
+                    ? _asset.metadata.name
+                    : _asset.metadata?.name + '#' + _asset.tokenId}
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTBasicInfo.tsx]: `undefined#111111`

File: packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx [hunk 25]
```diff
@@ -0,0 +1,66 @@
...
+const useStyles = makeStyles()((theme) => ({
+    wrapper: {
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        padding: 12,
+        boxSizing: 'border-box',
+        gap: 8,
+        borderRadius: 12,
+        background: '#F9F9F9',
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx]: Recommend to use `theme.palette.maskColor`.

File: packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx [hunk 26]
```diff
@@ -0,0 +1,66 @@
...
+interface NFTInfoCardProps {
+    asset: Web3Helper.NonFungibleAssetScope<void, NetworkPluginID.PLUGIN_EVM>
+}
+
+export function NFTInfoCard(props: NFTInfoCardProps) {
+    const { asset } = props
+    const { classes } = useStyles()
+    const { Others } = useWeb3State()
+    const infoConfigMapping = [
+        { title: 'Token ID', value: formatTokenId(asset.tokenId, 4) },
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTInfoCard.tsx]: Those title fields should support i18n.

File: packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx [hunk 27]
```diff
@@ -0,0 +1,72 @@
...
+
+    return (
+        <div className={classes.wrapper}>
+            <OpenSeaIcon width={24} height={24} />
+            <div className={classes.offerDetail}>
+                <div className={classes.flex}>
+                    <img className={classes.currencyIcon} src={offer.priceInToken?.token.logoURL} alt="" />
+                    <Typography className={classes.textBase}>
+                        <strong style={{ fontSize: 14 }}>{offer.priceInToken?.amount}</strong>{' '}
+                        <strong>{offer.price?.usd || '-'}</strong> - below
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx]: i18n

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 28]
```diff
@@ -0,0 +1,79 @@
...
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    const priceTokenImg = asset.priceInToken?.token.logoURL
+
+    return (
+        <div className={classes.wrapper}>
+            <div className={classes.header}>
+                <Typography className={classes.textBase}>{t('price')}</Typography>
+                <Typography className={classes.textBase}>
+                    {t('plugin_collectible_time_left')} <strong>-</strong> h <strong>-</strong> m
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: Is it WIP?

Comment (guanbinrui) [reply packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: Recommend to use `date-fns`. Maybe `formatDistanceToNow` is suitable here.

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx [hunk 29]
```diff
@@ -0,0 +1,92 @@
...
+                        <img width={18} height={18} src={topOrder.priceInToken?.token.logoURL} />
+                        <Typography className={classes.textBase}>
+                            <strong>{topOrder.priceInToken?.amount ?? '-'}</strong>
+                        </Typography>
+                        <Typography className={classes.textBase}>
+                            <strong>{topOrder.price?.usd ?? '-'}</strong>
+                        </Typography>
+                    </>
+                )}
+            </div>
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]:
```suggestion
            {topOrder && (
                <div className={classes.offerBox}>
                        <Typography className={classes.textBase}>{t('plugin_collectible_top_offer')}</Typography>
                        <img width={18} height={18} src={topOrder.priceInToken?.token.logoURL} />
                        <Typography className={classes.textBase}>
                            <strong>{topOrder.priceInToken?.amount ?? '-'}</strong>
                        </Typography>
                        <Typography className={classes.textBase}>
                            <strong>{topOrder.price?.usd ?? '-'}</strong>
                        </Typography>
                </div>
            )}
```

File: packages/mask/src/resources/GemIcon.tsx [hunk 30]
```diff
@@ -0,0 +1,82 @@
+import { SvgIconProps, SvgIcon } from '@mui/material'
+
+const svg = (
```
Comment (UncleBill) [inline packages/mask/src/resources/GemIcon.tsx]: Put it in `packages/icon/brands`. Suggest to put all these brand icons there.

File: packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx (hunk 28 above)
Comment (BillySh1) [reply packages/mask/src/components/shared/NFTCard/NFTPriceCard.tsx]: done

File: packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx (hunk 27 above)
Comment (BillySh1) [reply packages/mask/src/components/shared/NFTCard/NFTOfferCard.tsx]: done

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx [hunk 31]
```diff
@@ -0,0 +1,63 @@
...
+        color: theme.palette.maskColor.second,
+    },
+}))
+
+export function OffersTab() {
+    const { classes } = useStyles()
+    const { orders, provider } = CollectibleState.useContainer()
+    const _orders = orders.value?.data ?? EMPTY_LIST
+    const { t } = useI18N()
+    return useMemo(() => {
```
Comment (UncleBill) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx]: No need to use `useMemo` here, you can wrap with `memo`

```tsx
export const OffersTab = memo(function OffersTab() {
 //
})
```

Comment (BillySh1) [reply packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx]: ok

File: packages/mask/src/resources/GemIcon.tsx (hunk 30 above)
Comment (BillySh1) [reply packages/mask/src/resources/GemIcon.tsx]: there are some icons not provided, added todo

Comment (BillySh1) [reply packages/mask/src/resources/GemIcon.tsx]: done

**Review by guanbinrui — APPROVED** [review]

File: packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx (hunk 31 above)
Comment (UncleBill) [reply packages/mask/src/plugins/Collectible/SNSAdaptor/Tabs/OffersTab.tsx]: Don't forget to edit other tabs as well.

image

**Pull request merged**